rfc/rfc006-distributed-registry.md

```diff
@@ -279,6 +279,14 @@ When control over a DID document has been lost, the DID subject will have to hav
 Without Verifiable Credentials linked to the DID document, the DID document no longer has any value.
 The DID subject will have to go through the process of reacquiring all Verifiable Credentials for a new DID document.
 
+#### 3.3.4 Protection against public key zerging
+
+By registering a very large number of public keys in a DID Document an attacker can exhaust resources used for storing public keys.
+
+To prevent this from happening a DID Document MUST contain no more than 50 keys.
```

Comment:
50 feels a bit arbitrary. Perhaps limit the size of the network payload instead of the amount of keys?

Reply:
that would also be arbitrary.

A cluster of 5 nodes with their own keys would yield 5 keys. When rotating keys it would be doubled: 10.
So 10 is a well though minimum.
50 is a multiple of that and yes it's arbitrary.